README.md

```diff
@@ -1,18 +1,22 @@
 # DataJoint Element - DeepLabCut
 
-This repository features a DataJoint schema for pose estimation via ***DeepLabCut***.
++ This repository features a DataJoint schema for pose estimation via ***DeepLabCut***.
```

Comment:
Do we have a prescriptive design doc for what should be bullets vs. prose? My gut is that something > 2 sentences shouldn't be bulleted, so I left these as paragraphs. I would propose removing bullets for these first 2 paragraphs, but ignore if this contradicts design docs elsewhere

Reply:
Thanks @CBroz1.  We do not currently have a design doc.  I generally lean to using bullet points in READMEs, but am happy to apply your suggestions here.